src/EventListener/LogRevisionsListener.php

```diff
@@ -608,13 +612,18 @@ private function saveRevisionEntityData(EntityManagerInterface $em, ClassMetadat
     }
 
     /**
-     * @param array<string, mixed>  $assoc
-     * @param array<string, mixed>  $entityData
+     * @param array<string, mixed> $assoc
+     * @param array<string, mixed> $entityData
```

Comment:
You'll get a cs failure from this change

Reply:
I have made the indicated changes in the latest comitment